lib/screens/home_page.dart

```diff
@@ -107,6 +107,17 @@ class _HomePageState extends State<HomePage> {
     return filteredCourses;
   }
 
+  List<Course> filterCoursesByIterests(List interests, List<Course> courses) {
```

Comment:
ah it's back

Reply:
look in the class you'll understand there is an identical method